src/web3-utils.js

```diff
@@ -4,22 +4,46 @@
  * from this file.
  */
 import Web3 from 'web3'
+import BigNumber from 'bignumber.js'
```

Comment:
Unfortunately we're avoiding `bignumber.js` to avoid bundling two BN implementations (`BN.js` is faster and lighter, but doesn't support the nice rounding that comes with `bignumber.js` ;).